atom/browser/atom_permission_manager.cc

```diff
@@ -112,7 +112,7 @@ int AtomPermissionManager::RequestPermission(
     content::RenderFrameHost* render_frame_host,
     const GURL& requesting_origin,
     bool user_gesture,
-    const base::Callback<void(blink::mojom::PermissionStatus)>&
+    const base::RepeatingCallback<void(blink::mojom::PermissionStatus)>&
```

Comment:
ah... i changed this to `Once` in https://github.com/electron/electron/pull/18136

Reply:
ah that makes more sense :) will update!

Reply:
ah i'll just leave this as it was then